### PR TITLE
Pass Asset auths to pop-up

### PR DESCRIPTION
**Pull request by serge-fabiano**

### CODE REVIEW

**Comment by LucasBrazi06** [comment]
@Ramzay Could you check this PR and align everywhere all the pop-ups opened from all the lists and systematically pass the auth objects?
Thanks!

File: src/app/shared/table/actions/assets/table-create-asset-action.ts [hunk 1]
```diff
@@ -20,7 +21,8 @@ export class TableCreateAssetAction extends TableCreateAction {
     };
   }
 
-  private createAsset(assetDialogComponent: ComponentType<unknown>, dialog: MatDialog, refresh?: () => Observable<void>) {
-    super.create(assetDialogComponent, dialog, null, refresh);
+  private createAsset(assetDialogComponent: ComponentType<unknown>, dialog: MatDialog,
+    dialogParams: DialogParamsWithAuth<Asset, CarsAuthorizations>, refresh?: () => Observable<void>) {
```
Comment (Ramzay) [inline src/app/shared/table/actions/assets/table-create-asset-action.ts:25]: Same remark as above

File: src/app/shared/table/actions/assets/table-create-asset-action.ts [hunk 2]
```diff
@@ -1,14 +1,15 @@
...
 
-import { AssetButtonAction } from '../../../../types/Asset';
+import { Asset, AssetButtonAction } from '../../../../types/Asset';
 import { TableActionDef } from '../../../../types/Table';
 import { TableCreateAction } from '../table-create-action';
 
 export interface TableCreateAssetActionDef extends TableActionDef {
   action: (assetDialogComponent: ComponentType<unknown>, dialog: MatDialog,
-    refresh?: () => Observable<void>) => void;
+    dialogParams: DialogParamsWithAuth<Asset, CarsAuthorizations>, refresh?: () => Observable<void>) => void;
```
[the remark this answers is not in the document]
Comment (LucasBrazi06) [reply src/app/shared/table/actions/assets/table-create-asset-action.ts]: Fixed.
Thanks!

File: src/app/shared/table/actions/assets/table-create-asset-action.ts (hunk 1 above)
Comment (LucasBrazi06) [reply src/app/shared/table/actions/assets/table-create-asset-action.ts]: Fixed also.

**Comment by Ramzay** [comment]
> Could you make sure that we have the auths available in all pop-ups? It'll help when implementing new constraints like making a field mandatory in the UI.

I fully agree, will do it on previously migrated components

**Pull request merged**